### PR TITLE
[RAT-27] changes from the branch

### PR DESCRIPTION
**Pull request by Warrior1412**

### CODE REVIEW

**Comment by jdamerow** [comment]
- On the Consoritium > news & updates page (news-updates.html) are still login links
- everything in the "user" folder should not be needed anymore?
- the warning on the hps repository page isn't there

**Comment by Warrior1412** [comment]
[RAT-27] removed login from news-updates.html, removed contents in "user" folder should, warning on the hps repository

File: node/3.html [hunk 1]
```diff
@@ -106,9 +106,12 @@ <h2><a href="../index.html" title="Home">Digital|HPS</a></h2>
     <section id="post-content" role="main">
     <div id="breadcrumbs"><h2 class="element-invisible">You are here</h2><nav class="breadcrumb"><a href="../index.html">Home</a> » HPS Repository</nav></div>                <h1 class="page-title">HPS Repository</h1>                                          <span property="dc:title" content="HPS Repository" class="rdf-meta element-hidden"></span><span property="sioc:num_replies" content="0" datatype="xsd:integer" class="rdf-meta element-hidden"></span>  
       
-    
   <div class="node node-page node-full">
-    <div class="field field-name-body field-type-text-with-summary field-label-hidden"><div class="field-items"><div class="field-item even" property="content:encoded"><p>The <a href="http://hpsrepository.asu.edu/" target="_blank">HPS Repository</a> is a central database for storing and providing access to digital objects documenting the history and philosophy of science. The repository is where contributing users upload all of their objects for storage.  </p>
+    <div class="field field-name-body field-type-text-with-summary field-label-hidden"><!-- Warning Dialog Box -->
```
Comment (jdamerow) [inline node/3.html:110]: this is not visible

**Review by jdamerow — CHANGES_REQUESTED** [review]
And the login links are still there
image

File: node/3.html (hunk 1 above)
Comment (Warrior1412) [reply node/3.html:110]: Yes, it was because of "display: none".

**Comment by Warrior1412** [comment]
Addressed the comments and committed again. Please check it, if possible.

**Comment by jdamerow** [comment]
When you reopen the PR, it is automatically deployed and you can check (the link to the deployment is on the PR timeline)

**Comment by jdamerow** [comment]
Ok, one last thing I think, that I didn't notice before:
- https://digitalhps-org-pr-5.onrender.com/node/11.html: can you change the name to my name
- https://digitalhps-org-pr-5.onrender.com/node/9.html: can you change the email address to the digitalhps one.

Thanks!

**Comment by Warrior1412** [comment]
Done.

**Pull request merged**